tests/testthat/test-missing_argument_linter.R

```diff
@@ -18,14 +18,21 @@ test_that("missing_argument_linter skips allowed usages", {
 
 test_that("missing_argument_linter blocks disallowed usages", {
   linter <- missing_argument_linter()
-  lint_msg <- rex::rex("Missing argument in function call.")
+  lint_msg1 <- rex::rex("Missing argument 1 in function call.")
```

Comment:
"Missing positional argument 1" would read more clearly I think. Worth changing?

Reply:
I would leave it as is. I worry about beginners being confused if they don't know this term. also, it's somewhat ambiguous -- for `foo = function(x,y)` and call `foo(y = 1, )` is the first positional argument missing, or the second?

Reply:
Hmm okay. I was stumped because `1` also appeared (unrelatedly) in the line `foo(, a = 1)`.

Reply:
> Hmm okay. I was stumped because `1` also appeared (unrelatedly) in the line `foo(, a = 1)`.

I think the lint's column_number should help with that